uwrt_mars_rover_arm/uwrt_mars_rover_arm_hw/include/uwrt_mars_rover_arm_hw/uwrt_mars_rover_arm_hw_actuator.hpp

```diff
@@ -0,0 +1,147 @@
+#pragma once
+
+#include <bits/c++config.h>
+
+#include <cmath>
+#include <cstdint>
+#include <hardware_interface/actuator_interface.hpp>
+#include <hardware_interface/handle.hpp>
+#include <hardware_interface/hardware_info.hpp>
+#include <hardware_interface/types/hardware_interface_return_values.hpp>
+#include <memory>
+#include <rclcpp/logger.hpp>
+#include <rclcpp/macros.hpp>
+#include <rclcpp_lifecycle/state.hpp>
+#include <transmission_interface/simple_transmission.hpp>
+#include <vector>
+
+#include "uwrt_mars_rover_arm_hw/visibility.hpp"
+
+namespace uwrt_mars_rover_arm_hw
+{
+// namespaced structs to hold transmission data - state & command
+namespace TransmissionData
+{
+namespace ActuatorData
+{
+using CommandData = struct CommandData
+{
+  double velocity{};
+};
+
+using StateData = struct StateData
+{
+  double velocity{};
+  double position{};
+  double
+    iq_current{};  // actuator also provide iq_current - this is not needed for transmission however
```

Comment:
```suggestion
    iq_current{};  // unused in transmission
```

More succinct comments